src/schema/text_options.rs

```diff
@@ -14,6 +15,13 @@ pub struct TextOptions {
     indexing: Option<TextFieldIndexing>,
     #[serde(default)]
     stored: bool,
+    #[serde(default)]
+    #[serde(skip_serializing_if = "is_false")]
```

Comment:
```suggestion
```

Reply:
absence would be false, should be fine/intuitive for fastfields on string?

Reply:
Oh yes it is intuitive, but it has no real benefit and hurts discoverability.